baiji/serialization/yaml.py

```diff
@@ -14,20 +16,37 @@ def load(f, *args, **kwargs):
 
 
 def loads(s, *args, **kwargs):
-    import yaml
-    return yaml.load(s, *args, **kwargs)
+    return _load(s, *args, **kwargs)
 
 
 def dumps(obj, *args, **kwargs):
     import yaml
+    safe_dump = kwargs.pop('safe_dump', False) # Perhaps default should be True?
```

Comment:
Can we make this parameter `safe`, so it's the same name for both load and dump?